presto-main/src/main/java/com/facebook/presto/catalogserver/CatalogServer.java

```diff
@@ -163,4 +274,62 @@ private static String writeValueAsString(Object value, ObjectMapper objectMapper
             throw new RuntimeException(e);
         }
     }
+
+    private static boolean isCacheHit(CacheKey cacheKey, LoadingCache loadingCache)
+    {
+        if (loadingCache.getIfPresent(cacheKey) != null) {
+            return true;
+        }
+        return false;
+    }
+
+    private static CacheBuilder<Object, Object> newCacheBuilder(OptionalLong expiresAfterWriteMillis, long maximumSize)
+    {
+        CacheBuilder<Object, Object> cacheBuilder = CacheBuilder.newBuilder();
+        if (expiresAfterWriteMillis.isPresent()) {
+            cacheBuilder = cacheBuilder.expireAfterWrite(expiresAfterWriteMillis.getAsLong(), MILLISECONDS);
```

Comment:
We don't need `cacheBuilder =` here, do we?

Reply:
```suggestion
cacheBuilder.expireAfterWrite(expiresAfterWriteMillis.getAsLong(), MILLISECONDS);
```
This method returns itself. We don't need to assign it to itself again.